src/Digdir.Domain.Dialogporten.WebApi/Endpoints/V1/ServiceOwner/Dialog/GetDialogEndpoint.cs

```diff
@@ -34,3 +36,18 @@ public override async Task HandleAsync(GetDialogQuery req, CancellationToken ct)
             notFound => this.NotFoundAsync(notFound, ct));
     }
 }
+
+public sealed class GetDialogEndpointSummary : Summary<GetDialogEndpoint>
+{
+    public GetDialogEndpointSummary()
+    {
+        Summary = "Gets a single dialog";
+        Description = """
+                Gets a single dialog aggregate. For more information see the documentation (link TBD).
+                """;
+        Responses[StatusCodes.Status200OK] = "Successfully returned the dialog aggregate";
+        Responses[StatusCodes.Status401Unauthorized] = Constants.SummaryErrorServiceOwner401;
+        Responses[StatusCodes.Status403Forbidden] = "Unauthorized to get the supplied dialog (not owned by authenticated organization or has additional scope requirements defined in policy)";
+        Responses[StatusCodes.Status404NotFound] = "The given dialog ID was not found or is deleted";
```

Comment:
```suggestion
                Gets a single dialog aggregate. For more information see the documentation (link TBD).
                
                Note that it may return deleted dialogs.
                """;
        Responses[StatusCodes.Status200OK] = "Successfully returned the dialog aggregate.";
        Responses[StatusCodes.Status401Unauthorized] = Constants.SummaryErrorServiceOwner401;
        Responses[StatusCodes.Status403Forbidden] = "Unauthorized to get the supplied dialog (not owned by authenticated organization or has additional scope requirements defined in policy).";
        Responses[StatusCodes.Status404NotFound] = "The given dialog ID was not found.";
```

Merk her at det står: "Note that it may return deleted dialogs."